nni/retiarii/graph.py

```diff
@@ -617,7 +668,15 @@ def _dump(self) -> Any:
             ret['operation']['cell_name'] = self.operation.cell_name
         if self.label is not None:
             ret['label'] = self.label
+        if self.python_name is not None:
+            ret['python_name'] = self.python_name
         return ret
+    
+    def set_python_name(self, python_name: str):
```

Comment:
I think `node.python_name = xxx` is enough. Does not necessarily need to call `set_python_name`

Reply:
Thanks for the comment. Will refine it.